machAeroTutorials/overset_theory.rst

```diff
@@ -71,6 +117,41 @@ More about zipper meshes can be found here: `Enhancements to the Hybrid Mesh App
 Surface Loads Integration on Overset Structured Grids
 <https://www.nas.nasa.gov/assets/pdf/staff/Chan_W_Enhancements_to_the_Hybrid_Mesh_Approach_to_Surface_Loads_Integration_on_Overset_Structured_Grids.pdf>`_\.
 
+Collar Mesh
+===========
+
+We need collar meshes to represent intersections.
+
+.. figure:: images/overset_guide_10.jpg
+    :align: center
+
+    The collar mesh near the wing-strut intersection with the strut braced wing (SBW) configuration.
+..
+    src: overset_guide.pdf page 10
+
+We can also use a half-collar to reduce the number of overset blocks:
+
+.. figure:: images/overset_guide_11.jpg
+    :align: center
+
+    An example fuselage-horizontal tail region. The tail mesh is combined with the collar mesh for the fuselage.
+..
+    src: overset_guide.pdf page 11
+
+In this example, the half-collar on the fuselage belongs to the tail mesh.
+The half-collar and the rest of the tail mesh share the intersection line.
+
+
+Steps to Create an Overset Mesh
+===============================
+
+Here is a list of commonly used steps to create an overset mesh with the usual workflow in the MDO Lab:
+
+* Generate surface meshes for the main components in ICEM. 􏰀 Extrude surface meshes into volume meshes using pyHyp.
```

Comment:
I guess the second sentence is another bullet (which also explains the missing character there?)

Reply:
Yes! fixed